ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,4 +4,7 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
+  - enabled: 1
+    path: Assets/HoloToolkit-Examples/MotionControllers-GrabMechanics/Scenes/GrabMechanics.unity
```

Comment:
Nit: revert

Reply:
Do you want me to remove that scene from BuildSettings and recommit?

Reply:
Sorry about the late reply, but yes.

Reply:
Ok reverted EditorBuildSettings so there are 0 scenes and committed. Let me know if there's anything else.